errai-bus/src/main/java/org/jboss/errai/bus/server/servlet/StandardAsyncServlet.java

```diff
@@ -39,7 +39,7 @@
  * An implementation of {@link AbstractErraiServlet} leveraging asynchronous support of Servlet 3.0.
  *
  * @author Christian Sadilek <csadilek@redhat.com>
- * @author Mike Brock
+ * @author Mike BrockReflectionTests
```

Comment:
Not sure what happened here, but can you please remove this change?

Reply:
fat fingered :-)